aws-observability/templates/test/TestTemplate.yaml

```diff
@@ -1283,18 +1288,21 @@ Tests:
             - AutoEnableOptions.AutoSubscribeLambdaLogGroupsAWSResources.SumoLogGroupLambdaConnectorRole
             - sumoRdsMetricsAppStack.ClusterMetricRule
             - sumoRdsMetricsAppStack.InstanceMetricRule
+            - sumoApiGatewayMetricsAppStack.MetricRule
             - sumoAlbMetricsAppStack.AlbLogsFieldExtractionRule
             - sumoElbMetricsAppStack.ElbLogsFieldExtractionRule
             - sumoLambdaMetricsAppStack.LambdaCloudWatchLogsFieldExtractionRule
             - sumoLambdaMetricsAppStack.GenericCloudWatchLogsFieldExtractionRule
             - sumoNlbMetricsAppStack
             - sumoNlbMetricsAppStack.AddLoadBalancerField
             - sumoNlbMetricsAppStack.MetricRule
-            - sumoEC2MetricsAppStack.sumoAppCW
```

Comment:
Why did we delete this resource?

Reply:
As we set to 
```
 sumoEC2MetricsAppStack:
      Section2aInstallApp: 'No'
```
We don't required it

note: we are not installing any app in this testcase